lite/examples/object_detection/ios/ObjectDetection/Extensions/CVPixelBufferExtension.swift

```diff
@@ -26,8 +26,10 @@ extension CVPixelBuffer {
     let imageHeight = CVPixelBufferGetHeight(self)
 
     let pixelBufferType = CVPixelBufferGetPixelFormatType(self)
-
-    assert(pixelBufferType == kCVPixelFormatType_32BGRA)
+    
+    assert(pixelBufferType == kCVPixelFormatType_32BGRA ||
+           pixelBufferType == kCVPixelFormatType_32ARGB ||
+           pixelBufferType == kCVPixelFormatType_32RGBA)
```

Comment:
This change looks fine, but one question. 
Does the 32RGBA format actually work? Looks to me that only BGRA and ARGB formats are supported in the `rgbDataFromBuffer` in ModelDataHandler.

https://github.com/tensorflow/examples/blob/master/lite/examples/object_detection/ios/ObjectDetection/ModelDataHandler/ModelDataHandler.swift#L308

Should probably only allow BGRA and ARGB, and block RGBA. I'm not even sure in what cases the RGBA format is used.

What do you think?

Reply:
okay, I just updated the patch accordingly